mcs/build/profiles/winaot.make

```diff
@@ -28,6 +28,8 @@ PROFILE_MCS_FLAGS = \
 	-d:DISABLE_REMOTING \
 	-d:DISABLE_COM \
 	-d:WIN_PLATFORM \
+	-d:FEATURE_CONSOLE \
+	-d:NO_MULTIPLE_APPDOMAINS \
```

Comment:
And should we set it at all? I just figured multiple app domains aren't supported in the AOT case - if so I guess we should do the same for the testing full aot profile as well?